apps/data-backend/src/index.ts

```diff
@@ -1,35 +1,31 @@
-import express from "express";
-import dotenv from "dotenv";
-import router from "./router";
-import helmet from "helmet";
+import Fastify from "fastify";
+import fastifyCors from "@fastify/cors";
+import declareRoutes from "./router";
 import { launchBot, sendWebAppButton } from "./services/telegram-app";
-const cors = require("cors");
 
-dotenv.config();
-const app = express();
-
-app.use(helmet());
-app.use(
-  cors({
-    origin: process.env.APP_URL_WEB
-  })
-);
-app.use(express.json());
-app.use(express.urlencoded({ extended: true }));
+/**
+ * @type {import('fastify').FastifyInstance} Instance of Fastify
+ */
+const fastify = Fastify({
+  logger: true
+});
+fastify.register(fastifyCors, {
+  origin: "*",
+  methods: ["GET", "POST", "PUT", "DELETE"],
+  allowedHeaders: ["Content-Type", "Authorization"],
+  credentials: true
+});
+declareRoutes(fastify);
 
-app.use("/", router);
-const port = process.env.PORT || 5050;
-app.listen(port, () => {
-  console.log(`🚀 Backend server running at http://localhost:${port}`);
-  try {
-    launchBot(process.env.TELEGRAM_BOT_TOKEN || "");
-  } catch (error) {
-    console.error("Error launching bot:", error);
+fastify.listen({ port: 3000 }, function (err, address) {
```

Comment:
Can we use a process.env.PORT or a default value PORT to 5050, please?

Reply:
Okay